modules/apps/expando/expando-web/src/main/resources/META-INF/resources/edit/select_field_type.jsp

```diff
@@ -34,8 +34,40 @@ portletDisplay.setShowBackIcon(true);
 portletDisplay.setURLBack(redirect);
 
 renderResponse.setTitle(modelResourceName + ": " + ((expandoColumn == null) ? LanguageUtil.get(request, "new-custom-field") : expandoColumn.getName()));
+
+PortletURL customFieldURL = renderResponse.createRenderURL();
+
+customFieldURL.setParameter("mvcPath", "/view.jsp");
+customFieldURL.setParameter("redirect", redirect);
+
+PortalUtil.addPortletBreadcrumbEntry(request, LanguageUtil.get(request, "custom-field"), customFieldURL.toString());
```

Comment:
Since no params need to be set. We can just pass `String.valueOf(renderResponse.createRenderURL())` directly and save a var declaration.